recipes/pyjnius/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: pyjnius
+
+  {% set tagged_version = GIT_DESCRIBE_TAG %}
+  # If current revision is not tagged add number of commits since last tag to version.
+  {% if GIT_DESCRIBE_NUMBER|int != 0 %}
+    {% set tagged_version = tagged_version + '.post' + GIT_DESCRIBE_NUMBER %}
+  {% endif %}
+
+  version: {{tagged_version}}
+
+source:
+  git_url: https://github.com/kivy/pyjnius.git
```

Comment:
For now we need this, we have asked upstream for a new release. ( https://github.com/kivy/pyjnius/issues/342 ) The last one is too old (from a year ago).

Since there are some build issues that need to get sorted here, we are hoping to work on this asynchronously while the release is being sorted out. Hopefully we can incorporate it at the end.

Reply:
Should add that once we do have a release, we will be able to drop the Jinja above, which should fix the linter issue as well.

Reply:
We could take a git archive from an arbitrary sha.  Though i guess that doesn't differ much

Reply:
It's true. Not to worried about it as this should be transient anyways.